index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <title>JavaScript課題 ①-2</title>
+</head>
+<body>
+  <script>
+    let i = true
+    while (i) {
+    const result = window.prompt('日本の首都は？');
+    if (result === '知らん') {
+      //不適切な表現があったら処理を抜ける（アレンジしてみました）
+      i = false;
```

Comment:
ループを終了する場合は `break` を使います。
```suggestion
      break;
```